src/main/scala/uk/gov/hmrc/ct/ct600/v3/B295.scala

```diff
@@ -16,12 +16,13 @@
 
 package uk.gov.hmrc.ct.ct600.v3
 
-import uk.gov.hmrc.ct.box.{CtBoxIdentifier, CtInteger, Linked}
-import uk.gov.hmrc.ct.computations.CP264
+import uk.gov.hmrc.ct.box.{Calculated, CtBoxIdentifier, CtInteger, CtTypeConverters}
+import uk.gov.hmrc.ct.ct600.v3.retriever.CT600BoxRetriever
 
 case class B295(value: Int) extends CtBoxIdentifier(name = "Total of deductions and reliefs") with CtInteger
 
-object B295 extends Linked[CP264, B295] {
-
-  override def apply(source: CP264): B295 = B295(source.value)
+object B295 extends Calculated[B295, CT600BoxRetriever] with CtTypeConverters {
+  override def calculate(boxRetriever: CT600BoxRetriever): B295 = {
+    B295(boxRetriever.b275() + boxRetriever.b285())
```

Comment:
Does this change the expected value of B295? Also would the boxretriever in filing/frontend have to be changed to return the calculated value?